src/lib/mina_block/precomputed_block.ml

```diff
@@ -56,6 +56,7 @@ module T = struct
     ; staged_ledger_diff : Staged_ledger_diff.t
     ; delta_transition_chain_proof :
         Frozen_ledger_hash.t * Frozen_ledger_hash.t list
+    ; protocol_version : Protocol_version.t
```

Comment:
Should we be adding `proposed_protocol_version_opt` as well to precomputed blocks? I thought the idea of precomputed blocks is that we should be able to reconstruct the exact block again, and we technically need to know the proposed protocol version as well. Safe to assume any blocks produced so far have that set to `None`, but won't always be that way.

Reply:
Easy enough to add that, will do.